### PR TITLE
Patch issue 28

### PR DESCRIPTION
**Pull request by harikt**
Fixed issue #28 . ~~~Need to write tests.~~~

Some good read https://stackoverflow.com/questions/14610307/spl-arrayobject-arrayobjectstd-prop-list/16619183#16619183

### CODE REVIEW

**Comment by harikt** [comment]
@pmjones @koriym @jeremykendall thoughts ?

If anyone of you can look into this and approve I will merge and do a release.

Thank you.

**Comment by harikt** [comment]
Sorry missed to ping @jakejohns also.

File: src/Aura/Uri/Query.php [hunk 1]
```diff
@@ -19,6 +19,22 @@
...
 class Query extends \ArrayObject
 {
+
+    /**
+     *
+     * ArrayObject constructor used to setFlags
+     * @see https://github.com/auraphp/Aura.Uri/issues/28
+     *
+     * @param mixed $input
+     * @param integer $flags
```
Comment (koriym) [inline src/Aura/Uri/Query.php]: We normally use `int` instead of `integer` ?

**Comment by koriym** [comment]
@harikt Looks good to me. (Just noted small thing)

**Comment by harikt** [comment]
Thank you very much @koriym . Appreciate you taking time for the same.

File: src/Aura/Uri/Query.php (hunk 1 above)
Comment (harikt) [reply src/Aura/Uri/Query.php]: I will make the change. I did a search on this repo itself. Let it be consistent.

**Pull request merged**